lib/change_stream.js

```diff
@@ -90,7 +90,7 @@ class ChangeStream extends EventEmitter {
     }
 
     // Create contained Change Stream cursor
-    this.cursor = createChangeStreamCursor(this, options);
+    createChangeStreamCursor(this, options);
```

Comment:
This may be personal preference, but I prefer to assign value rather than mutate a passed in object when I can. I understand exceptions may rise. Thoughts?

Reply:
@emadum read you comment after I wrote this. What do you mean it shouldn't be immediately set, how is that different from what you're doing here?

Reply:
I wrote that it shouldn't NOT be immediately set. The code is functionally equivalent, I just think this way is less error-prone, in that you can't call `createChangeStreamCursor(this)` and forget to set it to `this.cursor`.

Reply:
I agree with @reggi here, unless we're considering renaming the function. Assigning the variable inside of this method feels like a side effect. I think a casual reader might consider L98 a bug because it's not very obvious while reading this method that `this.cursor` has ever been assigned.

Reply:
Yeah I called this out [here](https://github.com/mongodb/node-mongodb-native/pull/2310#discussion_r410708372). I can move it back to an explicit return/assignment if that's what everyone wants.

Reply:
yes, I think given it makes the assignment very obvious I prefer reverting this